angular-frontend/src/app/omnai-datasource/omnai-scope-server/live-data.service.ts

```diff
@@ -60,25 +64,37 @@ export class OmnAIScopeDataService implements DataSource{
 
   readonly #httpClient = inject(HttpClient);
 
+  private getAvailableDevices(): void {
+    timer(0, 15000)
+      .pipe(
+        switchMap(() => this.getDevices()),
+        takeUntil(this.destroy$)
+      )
+      .subscribe(devices => {
+        this.devices.set(devices);
+      });
+  }
 
   // Abrufen der verfügbaren Geräte vom Server
-  getDevices(): void {
+  public getDevices(): Observable<DeviceInformation[]> {
```

Comment:
I don‘t think this needs to be public.

This is a minor nitpick though. Otherwise the code looks pretty good to me